assets/js/phoenix_live_view.js

```diff
@@ -533,6 +533,10 @@ let DOM = {
         }
       },
       onBeforeElUpdated: function(fromEl, toEl) {
+        if (fromEl.isEqualNode(toEl)) {
+           return false // Skip this entire sub-tree
+        }
```

Comment:
I don't think this is a problem b/c of what the spec says about `isEqualNode` and deep comparisons including children and the index they are at - https://dom.spec.whatwg.org/#concept-node-equals